### ISSUE TITLE
Process after saving the checkpoint

### ISSUE THREAD

**Issue by fellow-tom** [ckkelvinchan/RealBasicVSR#23]
Hi. First of all, thank you very much for your project. The quality is impressive. I'm trying to train a neural network, but after every save checkpoint it starts some long process for 300 iterations. It looks like evaluating, but I couldn't find a value of 300 in the config file. I train on the REDS dataset (24k images) and that process takes longer than the training itself for 10k iterations. What is it? Is there any way to reduce this value (300)? Is it possible to disable it and what is the risk?

Example:
![screen](https://user-images.githubusercontent.com/99025833/154267857-6d9eee09-e4ec-48ca-b0b5-225f8c2a8575.jpeg)

**Comment by ckkelvinchan** [comment]
It is validation. What validation data did you use? Here 300 corresponds to 300 sequences.

**Comment by fellow-tom** [comment]
> It is validation. What validation data did you use? Here 300 corresponds to 300 sequences.

I used the REDS validation dataset (3k images). In my first attempt to train RealBasicVSR, I used this REDS validation dataset for both validation and training. It took much less time. When I downloaded the REDS training dataset for training, the validation process became very long. Now I wait 2 hours for 10k iterations of training and 12 hours of validation.

**Comment by ckkelvinchan** [comment]
If you use the official validation set. There should be 30 sequences instead of 300. I suspect that there is something wrong in the configuration file. Did you modify it?

**Comment by fellow-tom** [comment]
> If you use the official validation set. There should be 30 sequences instead of 300. I suspect that there is something wrong in the configuration file. Did you modify it?

I applied the command:
`python crop_sub_images.py --data-root ./data/REDS  --scales 4`
and the dataset became 30k instead of 3k

**Comment by ckkelvinchan** [comment]
I suggest using the provided configuration to test on UDM10. In this case you only need to test on 10 clips during validation. Also you can reduce the frequency of validation.

**Comment by fellow-tom** [comment]
> I suggest using the provided configuration to test on UDM10. In this case you only need to test on 10 clips during validation. Also you can reduce the frequency of validation.

Thank you very much. I'll give it a try.

**Comment by fellow-tom** [comment]
It helped. Thanks!
